scss/mixins/_table-variants.scss

```diff
@@ -1,15 +1,21 @@
-// Boosted mod: striped is not handled by table-variant()
+// Boosted mod: Variants have a brand different :hover and :striped management
 // scss-docs-start table-variant
 @mixin table-variant($state, $background) {
   .table-#{$state} {
     $color: color-contrast(opaque($body-bg, $background));
-    $hover-bg: mix($color, $background, percentage($table-hover-bg-factor));
-    $active-bg: mix($color, $background, percentage($table-active-bg-factor));
+    $hover-bg: mix($color, $background, percentage($table-variant-hover-bg-factor));
+    $striped-bg: mix($color, $background, percentage($table-variant-striped-bg-factor));
+    $striped-hover-bg: mix($color, $background, percentage($table-variant-striped-hover-bg-factor)); // Boosted mod
```

Comment:
If everything (or most) of this mixin is modified in Boosted, you can remove the Boosted mod comments within the mixin. Otherwise, they are not accurate compared to Bootstrap.